ui.js

```diff
@@ -78,80 +63,59 @@ const Search = ({query, emojis, skinNumber, selectedIndex, onChangeQuery}) => {
 	);
 };
 
-class Emoj extends React.PureComponent {
-	constructor(props) {
-		super(props);
-		autoBindReact(this);
-
-		this.state = {
-			stage: STAGE_CHECKING,
-			query: '',
-			emojis: [],
-			skinNumber: props.skinNumber,
-			selectedIndex: 0,
-			selectedEmoji: null
-		};
-	}
-
-	render() {
-		const {
-			stage,
-			query,
-			emojis,
-			skinNumber,
-			selectedIndex,
-			selectedEmoji
-		} = this.state;
-
-		return (
-			<Box>
-				{stage === STAGE_COPIED && <CopiedMessage emoji={selectedEmoji}/>}
-				{stage === STAGE_SEARCH && (
-					<Search
-						query={query}
-						emojis={emojis}
-						skinNumber={skinNumber}
-						selectedIndex={selectedIndex}
-						onChangeQuery={this.handleChangeQuery}
-					/>
-				)}
-			</Box>
-		);
-	}
+const Emoj = ({skinNumber: initialSkinNumber, onSelectEmoji}) => {
+	const {exit} = useApp();
+	const [stage, setStage] = useState(STAGE_CHECKING);
+	const [query, setQuery] = useState('');
+	const [emojis, setEmojis] = useState([]);
+	const [skinNumber, setSkinNumber] = useState(initialSkinNumber);
+	const [selectedIndex, setSelectedIndex] = useState(0);
+	const [selectedEmoji, setSelectedEmoji] = useState(null);
```

Comment:
Is it possible to use `undefined` here instead of `null`? I'm trying to abolish `null` from my projects.